geometry2/164cfa3/164cfa3.bug

```diff
@@ -88,4 +88,4 @@ fix:
 
   languages: C++
 
-  time: 2014-05-12 (21:11 GMT+1)
+  time: 2014-05-12T21:11:00Z GMT+1
```

Comment:
timezone is borked

Reply:
Please take a look at the discussion above. After my initial `awk` command, I have dropped the commit https://github.com/robust-rosin/robust/commit/0c4670703dfd75ccd4e25e0333b639c3e229917d in https://github.com/robust-rosin/robust/pull/359. Once it is merged, I will rebase this PR. Currently in this PR I have only worked on fault and failure codes.